### PR TITLE
[Oy2-20679] Feature Variant: USA Header Dark Variant

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Adds a dark variant to the UsaBanner. Looks like this:

Screen Shot 2022-09-13 at 2 46 28 PM

## Type of Request

feature

## Approach

Add a prop `variant` that when set to `"dark"` adds a class `usa-banner--dark` to the banner component. This new class adds the dark variant styles.

#### Pull Request Creator Checklist

- [X] It is documented what type of request this is.
- [X] There is a detailed description for the purpose of this PR.
- [X] This PR is in line with the intention of this Project

### CODE REVIEW

File: src/components/Radio/Radio.scss [hunk 1]
```diff
@@ -1,26 +1,27 @@
 @use "../../assets/theme/macbis-colors" as c;
 
-.usa-radio__input--tile+[class*=__label] {
+.usa-radio__input--tile + [class*="__label"] {
```
Comment (charmcitygavin) [inline src/components/Radio/Radio.scss:3]: Just curious, does SCSS freak out if you don't include these spaces?

File: src/components/UsaBanner/UsaBanner.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@use "../../assets/theme/macbis-colors" as c;
+
+// TODO: These colors do not exist in _macbis-colors
+// How and do we incorporate these? Should these be converted to existing colors?
+$black: #1b1b1b;
```
Comment (charmcitygavin) [inline src/components/UsaBanner/UsaBanner.scss:5]: Would the `base` color in the Figma (`#212121`) work in place of this color? There also appear to be a few gray options in that palette as well but I don't know how funky they'd look in this context.

**Review by charmcitygavin — APPROVED** [review]
Looks good to me! There may be a few colors in [the Figma](https://www.figma.com/file/faCD5JG364kC3CBX8YvnV2/MACBIS-Design-System) that you could potentially use for the black, gray, and white colors (see the "Style/Color" page in that doc).

File: src/components/Radio/Radio.scss (hunk 1 above)
Comment (jphechter) [reply src/components/Radio/Radio.scss:3]: I didn't intentionally make changes to this file. Prettier must have gotten a hold of that. I can undo this.

File: src/components/UsaBanner/UsaBanner.scss (hunk 2 above)
Comment (jphechter) [reply src/components/UsaBanner/UsaBanner.scss:5]: That's something I want to talk to Michael about, and that's really want my comment on line 4 is meant to address.

Given that this is a net-new design I want to honor what he's got here. If we're going to expand the color palate I want to know how we should be treating these colors before adding them to the macbis-colors stylesheet.

**Review by jdinh8124 — APPROVED** [review]

File: src/components/UsaBanner/UsaBanner.scss (hunk 2 above)
Comment (jphechter) [reply src/components/UsaBanner/UsaBanner.scss:4]: Confirmed with Michael this morning that we're going to keep the colors used in the design. This is most likely a one-off and if we have make changes in the future we'll cross that bridge. Updated comment to reflect the usage.

**Pull request merged**